docs/index.md

```diff
@@ -128,6 +128,7 @@ Now you can access the server at `http://localhost:8080`.
 
 - CoronaBot Deutschland [https://twitter.com/CoronaBot_DEU](https://twitter.com/CoronaBot_DEU)
 - Fallzahlen Aktuell - Inzidenz & mehr aus DE [https://play.google.com/store/apps/details?id=com.kokoschka.michael.casestoday&hl=de](https://play.google.com/store/apps/details?id=com.kokoschka.michael.casestoday&hl=de)
+- ImpfPush - Benachrichtigung über aktuelle Inzidenz-Zahlen aus Ihrer Region (https://www.impfpush.de)
```

Comment:
Hi, you have to add
`[https://www.impfpush.de]`
bevor 
`(https://www.impfpush.de)`

Reply:
Fixed :)